### ISSUE TITLE
liveServer.settings.proxy does't work for me

### ISSUE THREAD

**Issue by R10burning** [ritwickdey/vscode-live-server#605]
env: 
    win10 
    version  5.6.1
setting.json:
   liveServer.settings.proxy": {
    "enable": true,
    "baseUri": "/api",
    "proxyUri": "http://baobab.kaiyanapp.com"
  },
request: 
    axios.get('/api/v4/tabs/selected').then(function(res) {
          console.log(res)
        })
result: 404
Request URL: http://127.0.0.1:5500/api/v4/tabs/selected
Request Method: GET
Status Code: 404 Not Found

**Comment by valleylmh** [comment]
me too, proxy doesn't work

**Comment by bujidao** [comment]
I've tried and is works for me!

vscode config:
![image](https://user-images.githubusercontent.com/22849423/105132990-d1b33680-5b26-11eb-81e3-68f588f96c17.png)

you request:
![image](https://user-images.githubusercontent.com/22849423/105133128-1e970d00-5b27-11eb-9f4a-03a9684b2509.png)

**[node]: Keep in mind that after you changed you proxy configs, you need to restart you vscode, if not, your config won't works.**

**Comment by WeeHorse** [comment]
Nope, not working with post requests, only get requests work for me.

**Comment by chenweigh** [comment]
same problem.

```
"liveServer.settings.host": "localhost",
    "liveServer.settings.proxy": {
        "enable": true,
        "baseUri": "/vip",
        "proxyUri": "http://domain/vip"
    },
    "liveServer.settings.CustomBrowser": "chrome",
    "liveServer.settings.port": 8084,
    "liveServer.settings.root": "/",
    "liveServer.settings.useWebExt": false
```
$.ajax("/vip/get/")

404

**Comment by aszx0413** [comment]
Still not working

**Comment by ivanrodriguezd** [comment]
There is a solution for this issue? It's not working for me neither.
![Screenshot 2024-02-08 113131](https://github.com/ritwickdey/vscode-live-server/assets/42595160/6f0f2690-15f0-464b-a8ac-2726b05be1f6)

**Comment by FI-GGG** [comment]
> I've tried and is works for me!我已经尝试过并且对我有用！
> 
> vscode config: VSCode配置： ![image](https://user-images.githubusercontent.com/22849423/105132990-d1b33680-5b26-11eb-81e3-68f588f96c17.png)
> 
> you request: 您的请求： ![image](https://user-images.githubusercontent.com/22849423/105133128-1e970d00-5b27-11eb-9f4a-03a9684b2509.png)
> 
> **[node]: Keep in mind that after you changed you proxy configs, you need to restart you vscode, if not, your proxy won't works.[node]： 请记住，更改代理配置后，您需要重新启动 vscode，否则，您的代理将无法工作。**

必须重启vscode 才能用，You must restart vs code to use it
